dnf5/commands/download/download.cpp

```diff
@@ -71,13 +110,29 @@ void DownloadCommand::run() {
         package_query.resolve_pkg_spec(option->get_value(), {}, true);
         result_pset |= package_query;
     }
+    if (resolve_option->get_value()) {
+        for (auto pkg : result_pset) {
+            goal->add_rpm_install(pkg, settings);
+        }
+        auto transaction = goal->resolve(false);
```

Comment:
`goal->resolve(false);` -> `goal->resolve();`

Reply:
you need to rebase to the newest upstream/main. The `allow_erasing` parameter of `resolve` has been removed.

Reply:
Got it, thank you